ruby/vampires.rb

```diff
@@ -1,4 +1,4 @@
-# Ken's Vamps
+## Ken's Vamps
 ########################   Loop of employees ##########################
 puts "How many Employees will be processed? (Please enter a digit.)"
 processed = gets.chomp.to_i

```

Comment:
On line 32 Consider

with this setup you will have to change this file every year for it to work.

Check out the Ruby `Time` class. You should be able to handle a dynamic year check with something like `Time.new.year`